pkg/archive/jar.go

```diff
@@ -61,3 +62,43 @@ var StableJARBuildMetadata = ZipEntryStabilizer{
 		zf.SetContent(buf.Bytes())
 	},
 }
+
+var StableJAROrderOfAttributeValues = ZipEntryStabilizer{
+	Name: "jar-attribute-value-order",
+	Func: func(zf *MutableZipFile) {
+		if !strings.HasSuffix(zf.Name, "META-INF/MANIFEST.MF") {
+			return
+		}
+		r, err := zf.Open()
+		if err != nil {
+			return
+		}
+		manifest, err := ParseManifest(r)
+		if err != nil {
+			return
+		}
+		for _, attr := range []string{
+			"Export-Package",
```

Comment:
I know I didn't do so above but is there any resource or description for these keys that could help us understand the field semantics? I trust that these are all comma-separated but having some source to link back to would be really nice.

Reply:
(it's also fine if there aren't good docs to link to and we just mention that in a comment)

Reply:
[`6b2b11c` (#339)](https://github.com/google/oss-rebuild/pull/339/commits/6b2b11c0c636f6263ca18a54656a203694a2a9b3)

Reply:
Found another related issue: https://issues.apache.org/jira/browse/FELIX-6602
Documenting here to aggregate as much information as possible.

Reply:
Terrific!